### ISSUE TITLE
Show window on all workspaces?

### ISSUE THREAD

**Issue by colemickens** [swaywm/sway#2742]
Hi, this is my first time using a window manager for more than a few hours, and I'm not sure if my request fits into the paradigm of tiling window managers or not. If not, please feel free to just close this out.

I often work with some sort of video playing, for one reason or another. In conventional DEs, I can pin a windows to appear on all workspaces.

Is there an equivalent with Sway or tiling window managers in general? Maybe just for floating windows? (since you don't have to worry about the window layout on each workspace being potentiall different) ?

**Comment by RyanDwyer** [comment]
Yep, it's called floating windows with sticky enabled. Sticky makes it show on top of whatever workspace is active on that output.

I also recommend using mpv as the media player, as it has no UI.

**Comment by xeruf** [comment]
The binding works, but how do I configure this by default for a certain window?
This did not work:

    for_window [app_id="dragon_drop"] sticky toggle

**Comment by xeruf** [comment]
Or how would I configure it for all floating windows by default?

**Comment by bl4ckb0ne** [comment]
Refer to the wiki or the irc channel for configuration issues
